qiskit/utils/run_circuits.py

```diff
@@ -277,7 +277,13 @@ def run_qobj(
         if is_local_backend(backend):
             max_circuits_per_job = sys.maxsize
         else:
-            max_circuits_per_job = backend.configuration().max_experiments
+            backend_version = getattr(backend, "version", None)
```

Comment:
Same here. Change `backend_version` to `backend_interface_version`.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7563/commits/363e8dd699bd099f54b18201c1217424e5383be8